### PR TITLE
Removed offers dashboard

### PR DESCRIPTION
**Pull request by mllocs**

### CODE REVIEW

**Comment by mllocs** [comment]
[Eliminar offers dashboard](https://trello.com/c/bVHzZ0D2/151-eliminar-offers-dashboard)

File: app/views/application/_navbar.html.erb [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
           <ul class="nav nav-pills actions-menu">
             <%= render 'application/menus/user_list_link' %>
-            <%= render 'application/menus/offers_dashboard_link' %>
+            <%= render 'application/menus/offers_link' %>
```
Comment (markets) [inline app/views/application/_navbar.html.erb]: Maybe `offers_list_link` for consistency with the other partials: `user_list_link`, `inquiries_list_link`

File: app/views/offers/index.html.erb [hunk 2]
```diff
@@ -36,9 +36,6 @@
             <span class="caret"></span>
           </a>
           <ul class="dropdown-menu" role="menu">
-            <li>
-              <%= link_to "× #{t 'global.all'}", dashboard_offers_path %>
-            </li>
```
Comment (markets) [inline app/views/offers/index.html.erb]: I'd leave this link, as we also have in inquiries list. It's useful when you want to remove current search by category.

**Comment by markets** [comment]
Closes #440
Closes #321

**Comment by enricostano** [comment]
Deployed to staging

**Comment by sseerrggii** [comment]
It works fine! :+1:

**Pull request merged**